api/v1alpha1/vspherevalidator_types.go

```diff
@@ -14,19 +14,25 @@ type VsphereValidatorSpec struct {
 
 	Auth                           VsphereAuth                          `json:"auth" yaml:"auth"`
 	Datacenter                     string                               `json:"datacenter" yaml:"datacenter"`
-	EntityPrivilegeValidationRules []EntityPrivilegeValidationRule      `json:"entityPrivilegeValidationRules" yaml:"entityPrivilegeValidationRules"`
-	RolePrivilegeValidationRules   []GenericRolePrivilegeValidationRule `json:"rolePrivilegeValidationRules" yaml:"rolePrivilegeValidationRules"`
-	TagValidationRules             []TagValidationRule                  `json:"tagValidationRules" yaml:"tagValidationRules"`
-	ComputeResourceRules           []ComputeResourceRule                `json:"computeResourceRules" yaml:"computeResourceRules"`
+	EntityPrivilegeValidationRules []EntityPrivilegeValidationRule      `json:"entityPrivilegeValidationRules,omitempty" yaml:"entityPrivilegeValidationRules,omitempty"`
+	RolePrivilegeValidationRules   []GenericRolePrivilegeValidationRule `json:"rolePrivilegeValidationRules,omitempty" yaml:"rolePrivilegeValidationRules,omitempty"`
+	TagValidationRules             []TagValidationRule                  `json:"tagValidationRules,omitempty" yaml:"tagValidationRules,omitempty"`
+	ComputeResourceRules           []ComputeResourceRule                `json:"computeResourceRules,omitempty" yaml:"computeResourceRules,omitempty"`
+	NTPValidationRules             []NTPValidationRule                  `json:"ntpValidationRules,omitempty" yaml:"ntpValidationRules,omitempty"`
 }
 
 type VsphereAuth struct {
 	SecretName string `json:"secretName" yaml:"secretName"`
 }
 
+type NTPValidationRule struct {
+	Name        string   `json:"name" yaml:"name"`
+	ClusterName string   `json:"clusterName,omitempty" yaml:"clusterName,omitempty"` // if cluster name is specified, the host will be looked up under the cluster hierarchy
+	Hosts       []string `json:"hosts" yaml:"hosts"`
+}
 type ComputeResourceRule struct {
 	Name                         string                        `json:"name" yaml:"name"`
-	ClusterName                  string                        `json:"clusterName,omitempty" yaml:"clusterName"`
+	ClusterName                  string                        `json:"clusterName,omitempty" yaml:"clusterName"` // if cluster name is specified, the entity will be looked up under the cluster hierarchy
```

Comment:
Should be something like this:
```golang
// ClusterName is required when the vCenter Host(s) reside beneath a Cluster in the vCenter object hierarchy
ClusterName    string    `json:"clusterName,omitempty" yaml:"clusterName"`
```
Providing the comment above the field and starting it with `ClusterName` (the field name) it improves the autogenerated godocs.